core/lib/refinery/core/configuration.rb

```diff
@@ -32,6 +32,7 @@ module Core
     self.visual_editor_javascripts = []
     self.visual_editor_stylesheets = []
     self.plugin_priority = []
+    self.regex_white_list = false
```

Comment:
I still want to know whether we can support this in Loofah with existing config options